packages/docs/integrations/channels/amazonses.mdx

```diff
@@ -0,0 +1,196 @@
+
+[//]: # (TODO: Need an image of the SES section in dittofeed)
+
+Enter the access key, secret access key, and region into the appropriate fields in Dittofeed.
+
+### Setting up event forwarding
+
+Event forwarding allows Amazon SES to foward bounce, delivery, etc notifications to Dittofeed for tracking.
+
+Goto the SES section of the AWS console and click "Configuration Sets" on the sidebar.
+
+When you setup your SES account, an initial configuration set should have been created for you (usually named `my-first-configuration-set`).
+Click to open this configuration set.
+
+(If a configuration set was not created, you'll need to create one before proceeding)
+
+<Frame>
+![AmazonSES event destinations](/images/amazonses-eventdestinations.png)
+</Frame>
+
+Click the "Event Destinations" tab and "Add Destination"
+
+<Frame>
+![AmazonSES event types](/images/amazonses-eventtypes.png)
+</Frame>
+
+Check the following event types:
+
+* Rejects
+* Deliveries
+* Hard Bounces
+* Complaints
+* Subscriptions
+* Opens
+* Clicks
+
+Click "Next"
+
+<Frame>
+![AmazonSES event destination](/images/amazonses-eventdestination.png)
+</Frame>
+
+Select "Amazon SNS" for the destination type and enter a name for this destination.
+
+Make sure "Enabled" is checked.
+
+Click "Create SNS topic" in the next section.
+
+<Frame>
+![AmazonSES create SNS topic](/images/amazonses-createsnstopic.png)
+</Frame>
+
+Enter a topic name and display name.  Click "Create Topic"
+
+The new topic should be filled in on the destination screen.  Click "Next"
+
+<Frame>
+![AmazonSES destination review](/images/amazonses-destinationreview.png)
+</Frame>
+
+Review the information for the destination and click "Add Destination"
+
+We also need to associate this configuration set with one of our verified identities. Click "Verified Identities"
+on the sidebar and click the identity you'll be using with Dittofeed.
+
+Note: You can use multiple identities, just make sure to assign the configuration set to each.
+
+<Frame>
+![AmazonSES default configuration set](/images/amazonses-defaultset.png)
+</Frame>
+
+Click the "Configuration Set" tab and "Edit" to set the default configuration set to the one we recently created.
+
+Final step: Forward SNS messages to Dittofeed's webhook.
+
+Visit the SNS service page in the AWS Console.
+
+Click "Subscriptions" in the sidebar then "Create Subscription"
+
+<Frame>
+![AmazonSES subscription](/images/amazonses-subscription.png)
+</Frame>
+
+Click the "Topic ARN" field and select the topic you created earlier.
+
+Select HTTPS for "Protocol"
+
+Enter `https://[mydittofeed]/api/webhooks/amazon-ses` replacing `[mydittofeed]` with the domain name of your dittofeed instance.
+
+Click "Create subscription".
+
+The subscription should be confirmed almost immediately.  If not, check that the URL was entered correctly and your instance is reachable from the open internet.
+
+Send a test email and check that new events populate in the events page.
+
```

Comment:
Could add a sentence here about finding and visiting the confirmation url in the API logs?